src/optimization/models/gridBusOpt.cpp

```diff
@@ -535,216 +535,192 @@ void gridBusOpt::remove (gridLinkOpt *lnk)
 
 
 
-void gridBusOpt::setAll (const std::string &type, std::string param, double val, gridUnits::units_t unitType)
+void gridBusOpt::setAll (const std::string &type, const std::string &param, double val, gridUnits::units_t unitType)
 {
-
-  if ((type == "gen") || (type == "generator"))
+    if ((type == "gen") || (type == "generator"))
     {
-      for (auto &gen : genList)
+        for (auto &gen : genList)
         {
-          gen->set (param, val, unitType);
+            gen->set (param, val, unitType);
         }
     }
-  else if (type == "load")
+    else if (type == "load")
     {
-      for (auto &ld : loadList)
+        for (auto &ld : loadList)
         {
-          ld->set (param, val, unitType);
+            ld->set (param, val, unitType);
         }
     }
-
 }
 
 // set properties
 void gridBusOpt::set (const std::string &param,  const std::string &val)
 {
-	if (param[0] == '#')
-    {
-      
-    }
-  else
+    if (param[0] != '#')
     {
-      gridOptObject::set (param, val);
+        gridOptObject::set (param, val);
     }
 }
 
 void gridBusOpt::set (const std::string &param, double val, units_t unitType)
 {
-  
-
-  if ((param == "voltagetolerance")||(param == "vtol"))
+    if ((param == "voltagetolerance")||(param == "vtol"))
     {
 
     }
-  else if ((param == "angleetolerance") || (param == "atol"))
+    else if ((param == "angletolerance") || (param == "atol"))
```

Comment:
Fixed all of these I found. I haven't checked other strings for typos very thoroughly yet